media/com_fabrik/js/tipsBootStrapMock.js

```diff
@@ -190,14 +218,23 @@ define(['jquery', 'fab/fabrik'], function (jQuery, Fabrik) {
                         .css({top: 0, left: 0, display: 'block'})
                         .appendTo(inside ? this.$element : document.body);
 
-                    pos = this.getPosition(inside);
-
-                    actualWidth = $tip[0].offsetWidth;
-                    actualHeight = $tip[0].offsetHeight;
-
-                    switch (inside ? placement.split(' ')[1] : placement) {
+                    var pos = this.getPosition(inside);
+                    var actualWidth = $tip[0].offsetWidth;
+                    var actualHeight = $tip[0].offsetHeight;
+                    var maxwidth = this.options.maxwidth>0 ? parseInt(this.options.maxwidth) : actualWidth;
+                    var xpos = parseInt(window.fabrikTipXpos);
+                    var tippos = inside ? placement.split(' ')[1] : placement;
+                    switch (tippos) {
                         case 'bottom':
-                            tp = {top: pos.top + pos.height, left: pos.left + pos.width / 2 - actualWidth / 2};
+                            leftpos = pos.left + pos.width / 2 - maxwidth / 2 ;
```

Comment:
Can we make this `var` on first use as well?

Reply:
Just did that for all 3 - $tip, tp, and leftpos and removed the line at the top of the function that initialized those 3.  (FYI - The first 2 were not my doing)
I'll keep that in mind in the future.